site/views/draft/popup-game.njk

```diff
@@ -66,108 +60,108 @@ aside:
-    <div class="boundary-container boundary-container-1">
-      <esl-d-popup-game trigger="::find(.game-popup-trigger)">
-        <esl-trigger
-          target="#game-popup-h-ltr"
-          mode="toggle"
-          class="game-popup-trigger"
-          style="top: 124px; left: 124px;">
-          🚀
-        </esl-trigger>
-      </esl-d-popup-game>
-    </div>
-  </div>
-  <esl-popup
-    id="game-popup-h-ltr"
-    class="popup-game popup-h"
-    position="top"
-    container=".boundary-container-1"
-    dir="ltr"
-    margin-arrow="35">
-    <div class="popup-content-wrapper">
-      <div class="popup-content"> 🇱 🇹 🇷 🐶 🐼</div>
-    </div>
-    <span class="esl-popup-arrow"></span>
-  </esl-popup>
-</section>
-<h2>Horizontal, direction: RTL</h2>
 <section class="row">
-  <div>
-    <div class="boundary-container boundary-container-2">
-      <esl-d-popup-game trigger="::find(.game-popup-trigger)">
-        <esl-trigger
-          target="#game-popup-h-rtl"
-          mode="toggle"
-          class="game-popup-trigger"
-          style="top: 124px; left: 124px;">
-          🚀
-        </esl-trigger>
-      </esl-d-popup-game>
-    </div>
-  </div>
-  <esl-popup
-    id="game-popup-h-rtl"
-    class="popup-game popup-h"
-    position="top"
-    container=".boundary-container-2"
-    dir="rtl"
-    margin-arrow="35">
-    <div class="popup-content-wrapper">
-      <div class="popup-content"> 🇷 🇹 🇱 🦝 🐱 </div>
-    </div>
-    <span class="esl-popup-arrow"></span>
-  </esl-popup>
-</section>
-<h2>Vertical</h2>
-<section class="row">
-  <div>
-    <div class="boundary-container boundary-container-3">
-      <esl-d-popup-game trigger="::find(.game-popup-trigger)">
-        <esl-trigger
-          target="#game-popup-v"
-          mode="toggle"
-          class="game-popup-trigger"
-          style="top: 124px; left: 124px;">
-          🚀
-        </esl-trigger>
-      </esl-d-popup-game>
-    </div>
+  <div class="col-12">
+    <uip-root>
+      <script type="text/html"
+              label="Horizontal"
+              uip-snippet
+              uip-snippet-js="js-snippet-popup-game">
+        <esl-d-popup-game trigger="::find(.game-popup-trigger)">
+          <esl-trigger
+            target="#game-popup"
+            mode="toggle"
+            class="game-popup-trigger">
+            🚀
+          </esl-trigger>
+        </esl-d-popup-game>
+        <esl-popup
+          id="game-popup"
+          class="popup-game popup-h"
+          position="top"
+          container=".uip-preview-inner"
+          dir="ltr"
+          margin-arrow="35">
+          <div class="popup-content-wrapper">
+            <div collapsed class="popup-content">
+              <a class="popup-g-hide-btn" href="javascript:void(0)" role="button">Collapse</a>
+              <lorem-ipsum class="popup-g-text" paragraphs=".03"></lorem-ipsum>
+              <a class="popup-g-continue-btn" href="javascript:void(0)" role="button">Expand</a>
+            </div>
+          </div>
```

Comment:
```suggestion
          <div class="m-2" style="max-width: 500px">
            <p class="popup-content-wrapper">
              Space exploration has always captured the imagination of humanity. <esl-trigger target="::next" class="btn-link expand-trigger" active-class="hide">Expand</esl-trigger>
              <esl-toggleable>
                Rockets, the vehicles propelling us beyond our planet, are marvels of engineering and science.
                The journey begins with a fiery launch, overcoming Earth's gravity.
                As rockets ascend, they stage-separate, shedding weight to become more efficient.
                Reaching space, they enable satellites to orbit, telescopes to peer into the cosmos, and missions to distant planets.
                <esl-trigger target="::parent" class="btn-link">Collapse</esl-trigger>
              </esl-toggleable>
            </p>
          </div>
```

Reply:
For sure we need to cleanup inline styles above, limit them as much as we can, and include into visible part of playground